scripts/cabal-format.sh

```diff
@@ -2,6 +2,23 @@
 
 set -euo pipefail
 
-for f in $(git ls-files -- '*.cabal'); do cabal format $f; done
+if [[ "${1:-format}" != "format" && "${1:-format}" != "check" ]]; then
```

Comment:
We should be able to reduce this duplication, right?
```suggestion
mode="${1:-format}"

if [[ "${mode}" != "format" && "${mode}" != "check" ]]; then
```

Reply:
Yes thanks!